guides/doc-Configuring_Load_Balancer/topics/Promoting_SCAP_Content_to_Clients.adoc

```diff
@@ -45,4 +45,4 @@ ifeval::["{build}" == "satellite"]
 * For more information about adding puppet modules to {ProjectServer}, see {BaseURL}puppet_guide/chap-red_hat_satellite-puppet_guide-adding_puppet_modules_to_red_hat_satellite_6[Adding Puppet Modules to {ProjectNameX}] in the _Puppet Guide_.
 endif::[]
 
-* For more information about Content Views, see {BaseURL}content_management_guide/managing_content_views#Managing_Content_Views-Registering_Systems_to_Environments_and_their_Content_Views[Managing Content Views] in the _Content Management Guide_.
+* For more information about Content Views, see {ContentManagementDocURL}Managing_Content_Views-Registering_Systems_to_Environments_and_their_Content_Views[Managing Content Views] in the _Content Management Guide_.
```

Comment:
https://docs.theforeman.org/master/Content_Management_Guide/index-foreman.html#Managing_Content_Views-Registering_Systems_to_Environments_and_their_Content_Views

not working for Foreman

Reply:
https://access.redhat.com/documentation/en-us/red_hat_satellite/6.8/html-single/content_management_guide/index#Managing_Content_Views-Registering_Systems_to_Environments_and_their_Content_Views

not working for Satellite

Reply:
Fixed